packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1239,6 +1249,8 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     _selectionOverlay = null;
     _focusAttachment.detach();
     widget.focusNode.removeListener(_handleFocusChanged);
+    WidgetsBinding.instance.removeObserver(this);
```

Comment:
Good catch, looks like we forgot to add this earlier...

Reply:
Yeah I lucked into noticing that!